cmd/acra-connector/acra-connector.go

```diff
@@ -122,15 +127,16 @@ func handleConnection(config *Config, connection net.Conn) {
 			Errorln("Can't connect to AcraServer")
 		return
 	}
-	defer acraConn.Close()
 
 	acraConnWrapped, err := config.ConnectionWrapper.WrapClient(config.ClientID, acraConn)
 	if err != nil {
 		log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCantWrapConnection).
 			Errorln("Can't wrap connection")
+		if err := acraConn.Close(); err != nil {
+			log.WithError(err).Errorln("Error on closing connection with AcraServer")
```

Comment:
It might be AcraServer or AcraTranslator.

Shall we change log message to "AcraServer/AcraTranslator" or (better) check AcraConnector's `mode` and use appropriate naming.